### DISCUSSION TITLE
Loras

### DISCUSSION THREAD

**Discussion by caphalorthrow** [ljleb/prompt-fusion-extension#45] [Q&A]
Hey, 
I was palying around with this extension and while i got some result and variation using normal prompts like dragon,fireball,house,table,fish... etc i culdn#t get it to work with my custom character Loras. 
Here it seems like i always get the exact same picture indiscriminate of what values i use.
Any idea why that my be?

cheers 
caph

**Comment by ljleb** [comment]
Hi! If you mean something like this:

```
[<lora:dragonball:1> : <lora:oilpainting:1> : 0.2, 0.6]
```

Then this is not going to work. Loras are stripped from the prompt before they reach the extension code. The above is equivalent to `<lora:dragonball:1> <lora:oilpainting:1> [ : : 0.2, 0.6]`.

If instead, you meant something like this:

```
<lora:dragonball:1,2>
```

Then this is not supported at the moment. This could be a cool idea, will look into implementing it if it is possible.

**Reply by caphalorthrow** [reply]
I was thinking about the first way. 
And i'm not quite sure if i fully understand what you meant with the second option, since the way it's written it would just increase the loras strength/weight or not?

**Reply by ljleb** [reply]
Yeah the second option would work like that, similar to how attention interpolation works.

I'm afraid it's not going to be possible to achieve the first option, as the loras are striped from the prompt when the code receives the text. The position of loras in the prompt does not change anything, only their presence and weight affects the output.
